ksqldb-streams/src/main/java/io/confluent/ksql/execution/streams/TableSuppressBuilder.java

```diff
@@ -91,6 +93,18 @@ public TableSuppressBuilder() {
             keySerde,
             valueSerde
         );
+
+    final Suppressed.StrictBufferConfig strictBufferConfig;
+    long maxBytes = queryBuilder.getKsqlConfig().getLong(KsqlConfig.KSQL_SUPPRESS_BUFFER_SIZE);
+
+    if (maxBytes == -1) {
```

Comment:
```suggestion
    if (maxBytes < 0) {
```
nit: sometimes people typo